src/subscription_manager/managercli.py

```diff
@@ -2894,7 +2894,7 @@ def _do_command(self):
             store = syspurposelib.get_sys_purpose_store()
             if store:
                 store.sync()
-        except (OSError, ConnectionException) as ne:
+        except (OSError, ConnectionException, RateLimitExceededException) as ne:
```

Comment:
`RateLimitExceededException` is a subclass of `ConnectionException`, so I believe there is no need for this addition.

Reply:
You are right. Good point. I'm closing this PR and I will update the BZ bug report.